pyracf/common/security_admin.py

```diff
@@ -79,11 +83,29 @@ def __init__(
             self.__replace_valid_segment_traits(replace_existing_segment_traits)
         if additional_secret_traits is not None:
             self.__add_additional_secret_traits(additional_secret_traits)
+        self.set_running_userid(run_as_userid)
+
+    # ============================================================================
+    # Run as Other User ID
+    # ============================================================================
+    def set_running_userid(self, new_userid: Union[str, None]) -> None:
+        if new_userid is None:
+            self.__running_userid = new_userid
+            return
+        if not isinstance(new_userid, str) or len(new_userid) > 8 or new_userid == "":
+            raise UserIdError(new_userid)
+        self.__running_userid = new_userid.upper()
+
+    def clear_running_userid(self) -> None:
+        self.__running_userid = None
```

Comment:
Do we need a `clear_running_userid` function or is it good enough to just document that `set_running_userid` can take a `None` for the userid to indicate that it is being unset?

Reply:
I agree it could be superfluous, but I thought that there could be use cases where clearing it explicitly may be preferred, whereas programmatically pulling a userid from an object but having `None` restore defaults felt appropriate.